assets/xml/objects/object_shn.xml

```diff
@@ -1,97 +1,125 @@
+        <Animation Name="gShootingGalleryManEmptyAnim" Offset="0xD3AC" /> <!-- Original name is "shn_wait". Unused and one frame long -->
+        <Animation Name="gShootingGalleryManHandsOnTableAnim" Offset="0xD9D0" /> <!-- Original name is "shn_wait01" -->
+        <Animation Name="gSwampShootingGalleryManHeadScratchLoopAnim" Offset="0xDFEC" /> <!-- Original name is "shn_wait02" -->
+        <Animation Name="gSwampGuideChinScratchAnim" Offset="0xE6C4" /><!-- Original name is "shn_wait03" -->
+
+        <!-- Shooting Gallery Man and Swamp Guide Limbs -->
+        <Limb Name="gShootingGalleryManWaistLimb" Type="Standard" EnumName="SHOOTING_GALLERY_MAN_LIMB_WAIST" Offset="0xE6E0" />
+        <Limb Name="gShootingGalleryManLeftThighLimb" Type="Standard" EnumName="SHOOTING_GALLERY_MAN_LIMB_LEFT_THIGH" Offset="0xE6EC" />
+        <Limb Name="gShootingGalleryManLeftShinLimb" Type="Standard" EnumName="SHOOTING_GALLERY_MAN_LIMB_LEFT_SHIN" Offset="0xE6F8" />
+        <Limb Name="gShootingGalleryManLeftFootLimb" Type="Standard" EnumName="SHOOTING_GALLERY_MAN_LIMB_LEFT_FOOT" Offset="0xE704" />
+        <Limb Name="gShootingGalleryManRightThighLimb" Type="Standard" EnumName="SHOOTING_GALLERY_MAN_LIMB_RIGHT_THIGH" Offset="0xE710" />
+        <Limb Name="gShootingGalleryManRightShinLimb" Type="Standard" EnumName="SHOOTING_GALLERY_MAN_LIMB_RIGHT_SHIN" Offset="0xE71C" />
+        <Limb Name="gShootingGalleryManRightFootLimb" Type="Standard" EnumName="SHOOTING_GALLERY_MAN_LIMB_RIGHT_FOOT" Offset="0xE728" />
+        <Limb Name="gShootingGalleryManTorsoLimb" Type="Standard" EnumName="SHOOTING_GALLERY_MAN_LIMB_TORSO" Offset="0xE734" />
+        <Limb Name="gShootingGalleryManLeftShoulderLimb" Type="Standard" EnumName="SHOOTING_GALLERY_MAN_LIMB_LEFT_SHOULDER" Offset="0xE740" />
+        <Limb Name="gShootingGalleryManLeftForearmLimb" Type="Standard" EnumName="SHOOTING_GALLERY_MAN_LIMB_LEFT_FOREARM" Offset="0xE74C" />
+        <Limb Name="gShootingGalleryManLeftHandLimb" Type="Standard" EnumName="SHOOTING_GALLERY_MAN_LIMB_LEFT_HAND" Offset="0xE758" />
+        <Limb Name="gShootingGalleryManRightShoulderLimb" Type="Standard" EnumName="SHOOTING_GALLERY_MAN_LIMB_RIGHT_SHOULDER" Offset="0xE764" />
+        <Limb Name="gShootingGalleryManRightForearmLimb" Type="Standard" EnumName="SHOOTING_GALLERY_MAN_LIMB_RIGHT_FOREARM" Offset="0xE770" />
+        <Limb Name="gShootingGalleryManRightHandLimb" Type="Standard" EnumName="SHOOTING_GALLERY_MAN_LIMB_RIGHT_HAND" Offset="0xE77C" />
+        <Limb Name="gShootingGalleryManHeadLimb" Type="Standard" EnumName="SHOOTING_GALLERY_MAN_LIMB_HEAD" Offset="0xE788" />
+
+        <!-- Shooting Gallery Man and Swamp Guide Skeleton -->
+        <Skeleton Name="gShootingGalleryManSkel" Type="Flex" LimbType="Standard" LimbNone="SHOOTING_GALLERY_MAN_LIMB_NONE" LimbMax="SHOOTING_GALLERY_MAN_LIMB_MAX" EnumName="ShootingGalleryManLimbs" Offset="0xE7D0" />
```

Comment:
I feel like since this is shared it should have a more encompassing name, but it's pretty hard to think of one. I suppose you could name it after the object...

Reply:
I wanted to avoid using the object name for this, since "shn" likely has no meaning to anyone not already familiar with this object. This at least clues you into the fact that the shooting gallery men and the swap guide share the same skeleton. But I suppose I could name it something like "gBurlyGuySkel"?

Reply:
Yeah, it's tricky. That's a pretty good name, actually, happy for other suggestions, though.

Reply:
Would I change limb and limb display lists to include that name too, if I switched to it? So the only thing referencing the shooting gallery and swamp guides would just be the heads?

Reply:
I think so, yes.